<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositcalc</class>
 <widget class="QDialog" name="depositcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_deposit_amount">
        <property name="text">
         <string>Сумма вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_deposit_amount"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_deposit_term">
        <property name="text">
         <string>Срок вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_deposit_term"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_rate">
        <property name="text">
         <string>Процентная ставка, % годовых</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_rate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QComboBox" name="comboBox_term">
        <item>
         <property name="text">
          <string>срок в днях</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>срок в месяцах</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="comboBox_currency">
        <item>
         <property name="text">
          <string>RUB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EUR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USD</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_capitalization">
        <property name="text">
         <string>Капитализация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_capitalization">
        <item>
         <property name="text">
          <string>нет</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ежедневно</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ежемесячно</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в 2 месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в квартал</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в 4 месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в полгода</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в год</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_sum">
        <property name="text">
         <string>Сумма пополнения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_sum"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_replenishment">
        <property name="text">
         <string>Пополнение вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_replenishment">
        <item>
         <property name="text">
          <string>нет</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ежемесячно</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в 2 месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в квартал</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в 4 месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в полгода</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>раз в год</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QPushButton" name="pushButton_result">
        <property name="text">
         <string>Результат</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_interest">
        <property name="text">
         <string>Начисленные проценты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_interest2">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_taxAmount">
        <property name="text">
         <string>Сумма налога</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_taxAmount2">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_total">
        <property name="text">
         <string>Сумма на вкладе к концу срока</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total2">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
